src/Router/Validators/PathValidator.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gacela\Router\Validators;
+
+use Gacela\Router\Entities\RouteParams;
+
+final class PathValidator
+{
+    public function __invoke(string $path): bool
+    {
+        return $this->isPathValid($path);
+    }
+
+    protected function isPathValid(string $path): bool
+    {
+        return $this->isPathEmpty($path) || $this->isPathValidFormat($path);
+    }
```

Comment:
What do you think about this instead?
```suggestion
    public function isValid(string $path): bool
    {
        return $this->isPathEmpty($path) 
            || $this->isPathValidFormat($path);
    }
```